ewah/dag_factories/dbt_dag_factory.py

```diff
@@ -25,7 +25,7 @@ def dbt_dags_factory(
     schema_name="analytics",  # see https://docs.getdbt.com/dbt-cli/configure-your-profile/#understanding-target-schemas
     keepalives_idle=0,  # see https://docs.getdbt.com/reference/warehouse-profiles/postgres-profile/
     dag_base_name="T_dbt_run",
-    schedule_interval: Union[str, timedelta] = timedelta(hours=1),
+    schedule_interval: Optional[Union[str, timedelta]] = None,
```

Comment:
Same as previous

Reply:
Addressed in the previous comment - changed it back to have the default value.